added notes about inheritance for dir & lang global attributes (#35637)
mdn/content: files/en-us/web/html/global_attributes/lang/index.md

</div>
```

## Inheritance

> If an element has no `lang` attribute, then it inherits it from its [parent node](/en-US/docs/Glossary/Node/DOM), which in turn may inherit it from its parent, and so on.

Reviewer: ```suggestion
If an element has no `lang` attribute, it will inherit the `lang` value set on its [parent node](/en-US/docs/Glossary/Node/DOM), which in turn may inherit it from its parent, and so on.
```
